src/components/LeafletMap.js

```diff
@@ -192,7 +197,7 @@ class LeafletMap extends Component {
     const siteNamesSizeResult = new Set(siteNamesArr).size === 1;
 
     //if all sites property are the same (equals to 1), return true
-    return coordinatesSizeResult && siteNamesSizeResult;
+    return coordinatesSizeResult || siteNamesSizeResult;
```

Comment:
I have a good reason to change it back even though the && may make more sense here. I just remember my intention was to bypass that it easier. For some clusters the coordinates of the sites have a very minor different in decimal values.
Example: 
- firstSite { 17.801, 52.321 }
- secondSite { 17.800, 52.321 }

So my first check for unique coordinates won't work, so I check their site_name. That's why the || is there. 

Other issue is some of the clusters doesn't zoom to MAX when click on them so the condition to check max zoom also don't work.

It will be easier to see on the map. 
The decision here is for better user experience.

Reply:
No sweat! This is fine 👍